Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM php:7.1-apache
+MAINTAINER Fork CMS <info@fork-cms.com>
+
+# Enable Apache mod_rewrite
+RUN a2enmod rewrite
+
+# Install the dependencies and PHP extensions needed
+RUN apt-get update && \
```

Comment:
Each RUN command creates a separate docker cache layer. So doing a separate RUN for each command creates a lot of cache layers (and extra space). 

Also, apt-get update and apt-get install commands should always be combined. 
https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#apt-get

So not sure if we should split it up?